src/main/java/Duke.java

```diff
@@ -1,10 +1,75 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final String GREET = "____________________________________________________________\n" +
+            " __\n" +
+            "('')  Hello! I'm MK.II Titan with feelings.\n" +
+            "// \\  What can I do for you?\n" +
+            "|_/| \n" +
+            "____________________________________________________________\n";
+
+    private static final String END = "____________________________________________________________\n" +
+            " __\n" +
+            "( \")  Bye! I said with a disappointed look.\n" +
+            "|| \\\n" +
+            "|_||\n" +
+            "____________________________________________________________";
+
+    private static final String LOGO = " _    _      _                          \n" +
+            "| |  | |    | |                         \n" +
+            "| |  | | ___| | ___ ___  _ __ ___   ___ \n" +
+            "| |/\\| |/ _ \\ |/ __/ _ \\| '_ ` _ \\ / _ \\\n" +
+            "\\  /\\  /  __/ | (_| (_) | | | | | |  __/\n" +
+            " \\/  \\/ \\___|_|\\___\\___/|_| |_| |_|\\___|";
+
+    private static final int INDEX_AFTER_DONE = 5;
+    private static final int INDEX_AFTER_TODO = 5;
+    private static final int INDEX_AFTER_DEADLINE = 9;
+    private static final int INDEX_AFTER_EVENT = 6;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String userInput;
+        boolean isActive = true;
+        Scanner in = new Scanner(System.in);
+        TaskManager taskManager = new TaskManager();
+
+        printGreeting();
+        while (isActive) {
+            userInput = readUserInput(in);
+            isActive = checkIfUserQuits(userInput);
+            handleUserInput(taskManager, userInput);
+        }
+        printGoodbye();
+    }
+
+    private static void printGreeting() {
+        System.out.println(LOGO);
+        System.out.println(GREET);
+    }
+
+    private static void printGoodbye() {
+        System.out.println(END);
+    }
+
+    private static String readUserInput(Scanner in) {
+        return in.nextLine();
+    }
+
+    private static boolean checkIfUserQuits(String userInput) {
```

Comment:
Perhaps the function name could change to shouldQuit or isDone? When the input is "bye", the program should quit.